content/20.treatments.md

```diff
@@ -540,19 +540,59 @@ Although exciting, the cost of vaccine manufacturing and electroporation may mak
+They are recognized by ribosomes in vivo and then translated and modified into functional proteins [@doi:10.3389/fimmu.2019.00594]. 
+The resulting intracellular viral proteins are displayed on surface MHC I proteins, provoking a strong CD8+ T cell response as well as a CD4+ T cell and B cell-associated antibody responses [@doi:10.3389/fimmu.2019.00594]. 
+
+Naturally, mRNA is not very stable and can degrade quickly in the extracellular environment or the cytoplasm. 
+The LNP covering protects the mRNA from enzymatic degradation outside of the cell [@doi:10.4155/tde-2016-0006]. 
+Codon optimization to prevent secondary structure formation and modifications of the poly-A tail as well as the 5’ untranslated region to promote ribosomal complex binding can increase mRNA expression in cells. 
+Furthermore, purifying out dsRNA and immature RNA with FPLC (fast performance liquid chromatography) and HPLC (high performance liquid chromatography) technology will improve translation of the mRNA in the cell [@doi:10.3389/fimmu.2019.00594; @doi:10.1080/14760584.2017.1355245].
+
+mRNA vaccines confer many advantages over traditional viral vectored vaccines and DNA vaccines. 
+In comparison to live attenuated viruses, mRNA vaccines are non-infectious and can be synthetically produced in an egg-free, cell-free environment, and thereby reducing the risk of a detrimental immune response in the host [@doi:10.1016/j.immuni.2020.03.007]. 
+Unlike DNA vaccines, mRNA technologies are naturally degradable and non-integrating, and they do not need to cross the nuclear membrane in addition to the plasma membrane for their effects to be seen [@doi:10.3389/fimmu.2019.00594].  
+Furthermore, mRNA viruses are easily, affordably, and rapidly scalable.
+
+mRNA vaccines have been developed for therapeutic and prophylactic purposes. 
+mRNA vaccines have not been licensed or commercialized; however, several agents for cancer and infectious diseases, including rabies, coronavirus, influenza, rabies, and cytomegalovirus, have shown promising immunization in animal models and are currently in clinical trials [@url:https://clinicaltrials.gov/ct2/show/NCT01669096?term=mRNA+vaccine&draw=7]. 
+Preclinical data from Pardi et al. identified effective antibody generation against full-length FPLC-purified influenza hemagglutinin stalk-encoding mRNA in mice, rabbits, and ferrets [@doi:10.1038/s41467-018-05482-0]. 
+Similar immunological responses for mRNA vaccines were observed in humans in Phase I and II clinical trials operated by the pharmaceutical-development companies Curevac and Moderna for rabies, flu, and zika [@doi:10.1080/14760584.2017.1355245]. 
+
 
 ##### Anticipated Mechanism
 
-Why it may be useful
+Positively charged bilayer LNPs carrying the mRNA attract negatively charged cell membranes, endocytose into the cytoplasm [@doi:10.4155/tde-2016-0006], and facilitate endosomal escape. 
+LNPs can be coated with modalities recognized and engulfed by specific celltypes. 
```

Comment:
This is definitely a big advantage of using LNPs! Perhaps it might be good to give an example here to show what you mean?

Reply:
(Also I think a space is missing between "cell types")